support-frontend/assets/pages/[countryGroupId]/oneTimeCheckout.tsx

```diff
@@ -356,6 +358,35 @@ function OneTimeCheckoutComponent({
 			});
 		}
 
+		if (paymentMethod === 'PayPal') {
+			const paymentResult = await postOneOffPayPalCreatePaymentRequest({
+				currency: currencyKey,
+				amount: finalAmount ?? 0,
```

Comment:
We could probably handle this further up, if we don't have a `finalAmount` we shouldn't even start any of this payment stuff. And we could remove the if statement on line 414

Reply:
Yes thats a good idea, have moved the `finalAmount` check to top of `FormOnSubmit`. removed the lower check and failback shown above. Thanks